packages/webapp/src/containers/Finances/constants.js

```diff
@@ -40,4 +40,5 @@ export const REVENUE_FORM_TYPES = {
 export const LABOUR_ITEMS_GROUPING_OPTIONS = {
   EMPLOYEE: 'EMPLOYEE',
   TASK_TYPE: 'TASK_TYPE',
+  TASK_ID: 'TASK_ID',
```

Comment:
Could you double-check that this is going to be used?

Reply:
Not used, lingering thing from a different approach! Removed it, thank you 🙂